frontend/app/src/composables/blockchain/use-account-addition-notifications.ts

```diff
@@ -82,7 +82,7 @@ export function useAccountAdditionNotifications(): UseAccountAdditionNotificatio
         address: account.address,
         list: listOfFailureText.join('\n'),
       }),
-      severity: Severity.INFO,
+      severity: Severity.WARNING,
```

Comment:
I am thinking that the severity in this actually depends on the data.

We probably need to identify the different cases and change the severity based on the status.

Reply:
okay gonna revert this and do this in different PR then